UITBU/CISHack-UITBU
# Idea4theworld

## Description

- The Blood Tracking application has been successfully developed and tested on various android devices.
- Screens are developed.
- UI's are developed and updated.
- SplashScreen added for the application.
- Built on the expo client then eject the app to get both Android and IOS applications from the same code base.

## Screenshots

- 
![images](https://user-images.githubusercontent.com/71460522/113927514-38cdb680-980b-11eb-8de6-9bd310c93ef5.png)


- 
![Screenshot_1617866316](https://user-images.githubusercontent.com/71460522/113984478-c4c1fb80-9868-11eb-8441-64bb44b918f8.png)





## Tests
### Manual test cases run
_For each manual test case, list the steps to test or reproduce the PR._

- First, make sure you have your environment ready for the running react-native application.
- Then checkout to this branch and pull the latest code into your local system.
- Then do a yarn or npm to install all the dependencies.
- Then open two terminals and run the command react-native start and react-native run-android. 

## Platform tested
- [ ] Large
- [ ] Tablet

## OS tested
